src/index.js

```diff
@@ -1 +1,55 @@
-'use strict';
+/* eslint-disable no-console */
+import express from 'express';
+import 'dotenv/config';
+import cors from 'cors';
+import { userRouter } from './routers/userRouter.js';
+import { WebSocket, WebSocketServer } from 'ws';
+import http from 'http';
+
+const createServer = () => {
+  const app = express();
+
+  app.use(
+    cors({
+      origin: process.env.CLIENT_HOST || 'http://localhost:7071',
+      credentials: true,
+    }),
+  );
+
+  app.use(express.json());
+  app.use(userRouter);
+
+  app.use((req, res, next) => {
+    res.status(404).send({ message: 'Page Not Found' });
+  });
+
+  const server = http.createServer(app);
+
+  const wss = new WebSocketServer({ server });
+
+  wss.on('connection', (socket) => {
+    console.log('New Client Connected');
+
+    socket.on('message', (message) => {
+      console.log(`Recieved message ${message}`);
+
+      wss.clients.forEach((clie) => {
```

Comment:
There is a typo in the variable name `clie`. It should be `client` for better readability and consistency.